### DISCUSSION TITLE
Adding '_blank' capabilities to custom wallet URIs

### DISCUSSION THREAD

**Discussion by hcwilk** [rainbow-me/rainbowkit#2037] [Ideas]
Hello! I'm looking to integrate with a wallet that actually redirects the user to another URL to manage their wallet. In v1 of RainbowKit (which we were on), we were able to target '_blank' to send the wallet to a new tab. 

I can't find a way to do this with the library now, so I'd love if anyone knows a way to do this, or if this could be implemented in the main library! All I need it for is the desktop connector.

Thanks!

**Comment by 0xmkh** [comment]
@hcwilk Ah yeah we don't have support for `_blank`. Is it possible to do similar to what we've done with [ledgerWallet](https://github.com/rainbow-me/rainbowkit/blob/main/packages/rainbowkit/src/wallets/walletConnectors/ledgerWallet/ledgerWallet.ts) and [bloomWallet](https://github.com/rainbow-me/rainbowkit/blob/main/packages/rainbowkit/src/wallets/walletConnectors/bloomWallet/bloomWallet.ts) ? Both of them are desktop wallets.

**Reply by hcwilk** [reply]
The wallet we're integrating with is actually fully in the browser, but they don't have a panel like MetaMask or Rabby. They just send the user to a URL where they manage their wallet. How it's set up now is if the user clicks on that wallet, the URL for our app is replaced with the wallet.

If there's no way to support opening a wallet in a new tab, we likely just won't include that wallet until there's _blank support or the wallet provider supports in-tab wallet management.

Thanks for your help!

**Reply by 0xmkh** [reply]
I see. We could look at supporting `_blank`. Which wallet are you trying to add ?

**Reply by hcwilk** [reply]
It's [this wallet](https://chromewebstore.google.com/detail/internet-money-crypto-wal/ckklhkaabbmdjkahiaaplikpdddkenic?pli=1)

**Reply by 0xmkh** [reply]
I'll take a look
